components/routes/github-authentication.js

```diff
@@ -1,52 +1,48 @@
-const debug = require('debug')('github-authentication:debug')
-const error = require('debug')('github-authentication:error')
-const axios = require('axios')
-const octokit = require('@octokit/rest')
-
-
-module.exports = function(webserver, controller) {
+const debug = require('debug')('github-authentication:debug');
+const error = require('debug')('github-authentication:error');
+const axios = require('axios');
 
+function githubAuth(webserver, controller) {
     debug('Configured /github/auth url');
     webserver.get('/github/auth', (req, res) => {
-        debug('entered on /github/auth')
-        const { code, access_token, token_type } = req.query
+        debug('entered on /github/auth');
+        const {
+            code,
+            accessToken,
+            tokenType,
+        } = req.query;
 
         if (code) {
             debug('Bot Code: ', code);
-                        // add OAuth code to db
-            axios.post(`https://github.com/login/oauth/access_token` +
-                        `?client_id=${process.env.GITHUB_CLIENT_ID}` +
-                        `&client_secret=${process.env.GITHUB_CLIENT_SECRET}` +
-                        `&code=${code}` +
-                        `&state=${process.env.GITHUB_STATE_TOKEN}` +
-                        `&redirect_uri=${process.env.GITHUB_REDIRECT_URI}`
-            )
-
-            .then(result => {
-                debug(result.data)
-                result.data.split('&').forEach(e => {
-                    debug(e.split('='))
-                })
-
-            }).catch(err => debug(err))
-
-
-        } else if (access_token && token_type) {
-            debug(access_token, token_type)
+            // add OAuth code to db
+            axios.post('https://github.com/login/oauth/access_token'
+        + `?client_id=${process.env.GITHUB_CLIENT_ID}`
```

Comment:
I'm pretty sure the `+` needs to follow the first line and then the second line should be indented. Don't super care one way or the other but want to make sure we agree on it.